tests/integrationv2/test_session_resumption.py

```diff
@@ -243,6 +244,7 @@ def test_tls13_session_resumption_s2n_client(managed_process, cipher, curve, cer
                 b'SSL_accept:SSLv3/TLS write certificate') == num_full_connections
 
 
+@pytest.mark.flaky(reruns=7, reruns_delay=2, condition=platform.machine().startswith("aarch"))
```

Comment:
What are the errors when these tests fail due to flakiness? It seems like the tests could be broken if they require 5 or 7 retries in order to succeed?

Reply:
Current working theory is that this is the test framework/interactions with subprocesses.  I bumped the timeout values from 5 seconds to 60, and then noticed that the retried tests count exactly matches the number of tests running for 60 seconds. So these are being timed out by pytest, unclear why.

Reply:
Interesting ok. I guess the concern is that if we make a change that causes one of these tests to become flaky, we likely won't notice it since they're retrying so many times. But I guess the extra retries only apply to arm, so as long as we continue running them in an environment with retries disabled we should notice a flaky regression.

Reply:
To be fair, we're running these now on x86 with [2 retires](https://github.com/aws/s2n-tls/blob/main/tests/integrationv2/tox.ini#L22), so there is no CI running these specific integrationv2 tests with 0 retires. I don't believe we're collecting retry metrics, which would be an interesting datapoint...